inst/doc/geoknife.R

```diff
@@ -4,6 +4,9 @@ options(continue=" ")
 options(width=60)
 library(knitr)
 library(geoknife)
+query <- geoknife::query
+`values<-` <- geoknife::`values<-`
+id <- geoknife::id
```

Comment:
yes, when I added the new things in suggests for the geotiff vignette, there are function collisions for these.